configuration/templates/package/debian/DEBIAN/postrm.tmpl

```diff
@@ -4,12 +4,12 @@ set -e
 case "$1" in
 	remove|purge)
 		echo -n "Removing service logs..."{{if eq .Domain "edge"}}
-		rm /var/log/thingsplex/{{.PackageName}}/* || true{{else}}
-		rm /var/log/futurehome/{{.PackageName}}/* || true{{end}}
+		rm -r /var/log/thingsplex/{{.PackageName}} || true{{else}}
+		rm -r /var/log/futurehome/{{.PackageName}} || true{{end}}
 		echo "...OK"
 		echo -n "Removing service configuration and data files..."{{if eq .Domain "edge"}}
-		rm /opt/thingsplex/{{.PackageName}}/data/* || true{{else}}
-		rm /var/lib/futurehome/{{.PackageName}}/* || true{{end}}
+		rm -r /opt/thingsplex/{{.PackageName}} || true{{else}}
```

Comment:
this also recursively removes the files and directories in the parent directory of the `data` directory. I'm not sure we want that?
```suggestion
		rm -r /opt/thingsplex/{{.PackageName}}/data || true{{else}}
```

Reply:
There is no risk that PackageName will be empty string. It's validated by the tool so you need to enter the name.

I am not sure, but maybe there is a reason why we remove only data folder and not everything.